.npmignore

```diff
@@ -1,11 +0,0 @@
-!dist/
```

Comment:
why did you delete the npmignore file?

Reply:
I wanted to stop pushing our sourcemap files to npm, so I tried doing it via npmignore, then I realized it was redundant ever since we're using the `files` field in package.json
Also [this](https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d) :)